runner/debug.js

```diff
@@ -27,7 +27,7 @@ const breakpoints = new Array(lines.length);
 let paused = true;
 const byg = Byg({
   lines: originalLines,
-  pause: () => { paused = true; },
+  pause() { paused = true; },
```

Comment:
This change is essentially meaningless and a lot of other code in the project follows the previous syntax anyway?

Reply:
`pause() { paused = true; },` uses object shorthand syntax introduced in ES6, being honest this syntax is shorter and cleaner to read also it implements implicit naming - automatically uses property name which can be helpful in debugging

Reply:
Yeah but using it in just this one place is largely meaningless and just a change to make a change? (imo no offence)

Reply:
nvm, If you allow me to perform that i can give a PR after changing this on each and every file + some more changes related to ES6.

Reply:
> also it implements implicit naming
`foo: () => { ... }` also does implicit naming.

Thanks, but I don't think the changes are necessary; it just seems like changing to change.